style-guide.md

```diff
@@ -127,3 +127,6 @@ See also:
 * Avoid racist metaphors,
   including anything that might be interpreted as a reference to skin colors
   or allusions to racialized historical events.
+* Talk about "people" rather than "users" when that's unambiguous.
+  When talking about the people using a particular system or program,
```

Comment:
```suggestion
  When talking about the people using a particular system,
```

Reply:
I don't feel strongly about this, but I included both because I feel like people don't necessarily understand "Chrome" or "curl" as a "system". They certainly are, and maybe we can expect the TAG to understand that, but it still feels clearer to mention both.

If, having considered this, you still prefer just "system", I'm happy with accepting this tweak.

Reply:
I prefer using fewer words where that is sufficient, but this was a very minor point.